WordPress/src/main/res/layout/reader_recommended_blog_item_new.xml

```diff
@@ -0,0 +1,104 @@
+        app:layout_constraintGuide_begin="@dimen/margin_extra_large" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_bottom"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintGuide_end="@dimen/margin_extra_large" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_start"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_begin="@dimen/margin_extra_large" />
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/site_name"
+        style="@style/ReaderTextView.Label.Medium"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="@dimen/margin_medium"
+        android:layout_marginStart="@dimen/margin_medium"
+        android:includeFontPadding="false"
+        android:textAppearance="?attr/textAppearanceSubtitle1"
+        app:layout_constraintBottom_toTopOf="@+id/site_url"
+        app:layout_constraintEnd_toStartOf="@id/site_follow_icon"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toEndOf="@id/site_icon"
+        app:layout_constraintTop_toTopOf="@id/site_icon"
+        tools:text="Legal Nomads" />
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/site_url"
+        style="@style/ReaderTextView.Site.Header.Caption.Url"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_extra_small"
+        android:includeFontPadding="false"
+        app:layout_constraintBottom_toBottomOf="@id/site_icon"
+        app:layout_constraintEnd_toEndOf="@id/site_name"
+        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintStart_toStartOf="@id/site_name"
+        app:layout_constraintTop_toBottomOf="@+id/site_name"
+        tools:text="legalnomads.com" />
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/site_description"
+        style="@style/ReaderTextView.Site.Description"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_medium"
+        android:includeFontPadding="false"
+        app:layout_constraintBottom_toBottomOf="@id/guideline_bottom"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@+id/site_icon"
+        tools:text="@tools:sample/lorem/random" />
+
+    <org.wordpress.android.ui.reader.views.ReaderFollowButton
+        android:id="@+id/site_follow_icon"
+        style="@style/Reader.Follow.Button.New"
```

Comment:
Regarding these steps:
7 - Repeat step 2 but this time disable the FF;
8 - Repeat steps 3-5 and verify the layout: the follow icon should appear like it currently does in prod;

The follow icon when the FF is OFF is smaller than the original prod version because the code that enlarges the icon when there's no caption in the old style was removed in a previous PR: https://github.com/wordpress-mobile/WordPress-Android/pull/19246/files#diff-6ef58bacdd65790941185baf4dab55cb62e9c6febb534b33ee36fec8159bcbf9L59

❓ Since that's the case and the `Reader.Follow.Button.Icon` is already different than the prod/old style (the icon is smaller) do we need to have this new layout just for using the new style?

I understand this layout is going to be used in another task, but I think the `Reader.Follow.Button.New` style could be used for the old `WordPress/src/main/res/layout/reader_recommended_blog_item.xml` file as well since the new button style is technically not behind the feature flag anyway. Thoughts?

Reply:
Good point 👍  since the "old" design is also using the new button design in other places, I've replaced `Reader.Follow.Button.Icon` for the "old" layout.